plugins/com.google.cloud.tools.eclipse.appengine.validation.test/src/com/google/cloud/tools/eclipse/appengine/validation/WebXmlValidatorPluginTest.java

```diff
@@ -51,30 +52,30 @@
 
   @Before
   public void setUp() throws CoreException {
-    // Project's default source folder is the main project folder.
     IProject project = projectCreator.getProject();
     javaProject = projectCreator.getJavaProject();
 
     resource = project.getFile("WebContent/WEB-INF/web.xml");
 
-    ValidationTestUtils.createFolders(project, new Path("src/main/java"));
-    IFile servletClass = project.getFile("src/main/java/ServletClass.java");
```

Comment:
There did not exist any physical files; they were being created here on the fly (via `IFile.create()`). I just refactored the repetitive file creation code fragments into the `createFile()` helper method.

To clarify, these on-the-fly files are nevertheless auto-deleted when a test is complete, as the project itself will be deleted. What this PR does is to manually delete some of these files (specifically, JSP files) in `tearDown()` before the project is auto-deleted, to prevent some JSP-specific build jobs from doing wonky things after the project is deleted.